ui/redux/selectors/livestream.js

```diff
@@ -87,6 +87,18 @@ export const selectActiveLivestreamForChannel = createCachedSelector(
   }
 )((state, channelId) => String(channelId));
 
+export const selectIsCurrentClaimLive = createCachedSelector(
+  (state, claimId) => claimId,
+  selectActiveLivestreamForClaimId,
+  (claimId, activeLivestreamForClaim) => {
+    if (!claimId || !activeLivestreamForClaim) {
+      return null;
+    }
+
+    return activeLivestreamForClaim.claimId === claimId || null;
+  }
+)((state, channelId) => String(channelId));
```

Comment:
I didn't try this out to be 100% sure, but I think `selectActiveLivestreamForClaimId` would be returning the claim for the given `channelId` anyway, so seems like it's going in circles?

Would using `selectActiveLivestreamForClaimId` and checking if there is a claim be sufficient for clients of `selectIsCurrentClaimLive`?

Reply:
If this is somehow needed, then just do some cleanup:
- activeLivestreamForClaim --> activeLivestreamClaim (or claim, or activeLivestreamClaimForId)
- selectIsCurrentClaimLive --> selectIsCurrentClaimLiveForId (keeping the ForXXX to make it clear it's a multi-param selector).